packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -28,17 +43,51 @@
             oc-child-width-1-3@s
           "
         >
-          <a v-for="space in spaces" :key="space.id" href="#" class="oc-mb-m">
-            <span class="spaces-list-card oc-border oc-card oc-card-default">
-              <span class="oc-card-media-top oc-border-b">
+          <div v-for="space in spaces" :key="space.id" class="oc-mb-m">
```

Comment:
If we're rendering a bunch of buttons (one for each space) a HTML list is perhaps more suitable than divs inside divs? 🤔